packages/plugin-coverage/src/lib/runner/lcov/lcov-runner.ts

```diff
@@ -76,13 +80,18 @@ export async function parseLcovFiles(
     }),
   );
   if (parsedResults.length !== results.length) {
-    throw new Error('Some provided LCOV results were not valid.');
+    const message = 'Some provided coverage LCOV results were not valid.';
+    if (continueFail) {
+      ui().logger.warning(message);
+    } else {
+      throw new Error(message);
+    }
   }
```

Comment:
Does this code ever run? :raised_eyebrow: 

The `parsedResults` are created from `await Promise.all(results.map(...))`, so how could `parsedResults` and `results` have different lengths?

Reply:
Good question, I'll findout.

Reply:
It was unreachable, refactored that code a bit